CMakeLists.txt

```diff
@@ -225,7 +225,10 @@ if(ENABLE_IPV6 AND NOT WIN32)
   endif()
 endif()
 
-curl_nroff_check()
+if(USE_MANUAL)
+    #nroff is currently only used when USE_MANUAL is set, so we can prevent the warning of no *NROFF if USE_MANUAL is not _FILE_OFFSET_BITS, by not even looking for NROFF..
```

Comment:
Hm, I suppose this error message got messed up somehow? :grinning:

Reply:
Sorry about that... autocomplete / autocorrect playing games.
Tidied up in new commit